kikuchipy/signals/ebsd_master_pattern.py

```diff
@@ -61,6 +74,145 @@ def __init__(self, *args, **kwargs):
         self.projection = kwargs.pop("projection", None)
         self.hemisphere = kwargs.pop("hemisphere", None)
 
+    def get_patterns(
+        self,
+        rotations: Rotation,
+        detector: EBSDDetector,
+        energy: int,
+        n_chunk: Optional[int] = -1,
+        dtype_out: Optional[np.dtype] = np.float32,
```

Comment:
The `Optional` type hint is for use when one valid option and None are the options for a parameter, not when there is a default value (https://docs.python.org/3/library/typing.html#typing.Optional). Here it should just be

```python
n_chunk: Optional[int] = None,
dtype_out: np.dtype = np.float32,
compute: bool = False,
```